tests/GUI/spaceBasicTests.spec.ts

```diff
@@ -19,150 +19,142 @@ test.describe(
     ],
   },
   async () => {
-    test(
-      "Basic test for checking if creating new space works correct",
-      {
-        tag: "@space",
-        annotation: {
-          type: "issue",
-          description: "https://github.com/marian51/poc-playwright-typescript-gui-api/issues/13",
+    let leftMenu: LeftMenu;
+    let newSpaceName: string;
```

Comment:
ustalamy, że robimy tak wszędzie, jeśli page się powtarzają w danym scope?

Reply:
Omówimy jutro na daily

Reply:
a nie lepiej to wyciągnąć po prostu nad describe pierwszy niż tak zagnieżdżone describe'y?